### PR TITLE
[US market]Leedarson zigbee3.0 dimmer bulb

### PR DESCRIPTION
**Pull request by FenMei007S**
@tpmanley @greens  this is for leedarson zigbee3.0 dimmer bulb. US market, plz deploy to global server

### CODE REVIEW

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -60,6 +60,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "innr", model: "RS 225", deviceJoinName: "Innr Smart Spot White"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-N13", deviceJoinName: "Sengled Element Classic"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-N14", deviceJoinName: "Sengled Element Classic"
+                fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FEDC", outClusters: "000A, 0019", manufacturer: "LDS", model: "ZBT-DIMLight-GLS0006", deviceJoinName: "Smart Bulb"
```
Comment (greens) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: please add the manufacturer's name to the join name

"Leedarson Smart Bulb"

**Comment by FenMei007S** [comment]
@greens updated

**Comment by FenMei007S** [comment]
@greens, Hm.. the bulb using the brand name is "smartthings", is it proper to show vendor name? Smart Bulb is the name get from HQ.

**Comment by greens** [comment]
Then I guess just change it back to "Smart Bulb". I was just using the name in the pull request title.

**Comment by FenMei007S** [comment]
@greens updated

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
Do you have a jira ticket number? I couldn't find one.

**Comment by FenMei007S** [comment]
The ticket not be registered yet, can it be merged without ticket?

**Pull request merged**